src/components/header.js

```diff
@@ -68,73 +66,70 @@ const Header = () => {
           </Link>
         </div>
         <div
-          sx={{
-            display: [`none`, `flex`],
-            width: `100%`,
-          }}
-        >
-          <SearchBar sx={{ width: `100%` }} />
-        </div>
-        <button
-          sx={{
-            display: [`flex`, `none`],
-            alignItems: `center`,
-            justifyContent: `center`,
-            borderRadius: `2`,
-            height: `39px`,
-            width: `39px`,
-            cursor: `pointer`,
-            outline: 0,
-            border: 0,
-            backgroundColor: lighten(`border`, 0.075),
-            '&:active, &:focus': {
-              boxShadow: theme => `0px 0px 0px 3px ${theme.colors.accent}`,
-            },
-          }}
-          onClick={() => navigate(`search`)}
+          sx={{ width: `max-content`, alignSelf: `flex-end`, display: `flex` }}
         >
-          <FiSearch size={20} />
-        </button>
-        {isAuthenticated() ? (
-          <div
-            sx={{
-              display: [`none`, `flex`],
-              alignItems: `center`,
-              width: `max-content`,
-              whiteSpace: `nowrap`,
-            }}
-          >
-            <Button
-              sx={{ variant: `buttons.primary`, height: `39px` }}
-              onClick={() => navigate('/palette/recipes')}
+          {isAuthenticated() ? (
+            <div
```

Comment:
I'm seeing this problem with the button: 

![image](https://user-images.githubusercontent.com/21114044/75491545-5aefd680-5973-11ea-83ba-f18e211f60f0.png)


Seems like the flex properties aren't being applied right.